cli.js

```diff
@@ -27,19 +36,34 @@ const cli = meow(`
 
 	Options
 	  --copy -c  Copy the first emoji to the clipboard
+	  --skin-tone -s  Set the default skin tone of the emoji (0 to 5)
 
 	Run it without arguments to enter the live search
+	Use Up/Down keys during live search to change the skin tones
 `, {
 	boolean: [
 		'copy'
 	],
 	alias: {
-		c: 'copy'
+		c: 'copy',
+		s: 'skinTone'
 	}
 });
 
+if (typeof cli.flags.skinTone === 'number') {
```

Comment:
It all depends on what @sindresorhus wants. At the moment he doesn't have any validation in place, that's why I suggest to not add it since it will become inconsistent (either all flags should have validation or none).

```
config.set('skinNumber', Math.max(0, Math.min(5, Number(cli.flags.skinTone)) || 0) ;
```